core/src/Cuda/Kokkos_Cuda_abort.hpp

```diff
@@ -56,21 +56,20 @@ extern "C" {
 /*  Cuda runtime function, declared in <crt/device_runtime.h>
  *  Requires capability 2.x or better.
  */
-extern __device__ void __assertfail(const void *message, const void *file,
-                                    unsigned int line, const void *function,
-                                    size_t charsize);
+[[noreturn]] __device__ void __assertfail(const void *message, const void *file,
+                                          unsigned int line,
+                                          const void *function,
+                                          size_t charsize);
 }
 
 namespace Kokkos {
 namespace Impl {
 
-__device__ inline void cuda_abort(const char *const message) {
-#ifndef __APPLE__
```

Comment:
Why did you remove that check?

Reply:
I'll fix this, sorry, I was just trying to make sure all the configurations in CI passed.

Reply:
Thanks!

Reply:
Fixed.